ethbloom/benches/unrolling.rs

```diff
@@ -1,73 +1,60 @@
-#![feature(test)]
+#![allow(clippy::cognitive_complexity)]
```

Comment:
well, clippy for benches seem ambitious but how would run you it in clippy? 

```$ cargo clippy --all-features```?

Since this is just a warning, I would rather prefer the warning instead of ignoring it. Thoughts?

Reply:
Okay, I will remove it.